Quote Generator/README.md

```diff
@@ -0,0 +1,11 @@
+## Quote Generator
+It was a simple Quote Generator app that generate new quotes after hitting button
+
+## Tech Stacks
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## Screenshots
+

```

Comment:
Screenshots Heading is there, but no Screenshot there 😓

Reply:
hehe I forgot to add😅